test/plugins/common.js

```diff
@@ -74,8 +75,10 @@ function cleanTraces(agent) {
   }
 
   var privateAgent = agent.private_();
-  privateAgent.traceWriter.buffer_ = [];
-  privateAgent._shouldTraceArgs = [];
+  if (privateAgent) {
+    privateAgent._shouldTraceArgs = [];
+  }
+  TraceWriter.get().buffer_ = [];
```

Comment:
A type checker would tell you that TraceWriter.get() can potentially return undefined. Now you haven't changed this code, but I think your implicit assumption is that TraceWrite.create() must have already been called?

If that's the case, shouldn't there be an assertion in `get()` to make sure it doesn't return undefined?